src/formats/MMTF.cpp

```diff
@@ -86,10 +86,14 @@ void MMTFFormat::read(Frame& frame) {
     frame.resize(0);
 
     if (structure_.unitCell.size() == 6) {
-        frame.set_cell(UnitCell(
-            structure_.unitCell[0], structure_.unitCell[1], structure_.unitCell[2],
-            structure_.unitCell[3], structure_.unitCell[4], structure_.unitCell[5]
-        ));
+        frame.set_cell({
+            static_cast<double>(structure_.unitCell[0]),
```

Comment:
``structure_.unitCell`` is already a vector of doubles. Why are the ``static_cast``s needed?

Reply:
I am pretty sure I got a warning without them. I'll check again.

Reply:
UnitCell is defined as a vector of float, see https://github.com/chemfiles/mmtf-cpp/blob/63e58a9057ee684b5a0a409932e7fc2126894fc0/include/mmtf/structure_data.hpp#L154

Reply:
Odd, Ok.